e2e/katz_online_test/test_online_call_get_and_set/input.cyp

```diff
@@ -0,0 +1,38 @@
+setup: |-
+    CREATE TRIGGER katz_online BEFORE COMMIT EXECUTE CALL katz_centrality_online.update(createdVertices, createdEdges, deletedVertices, deletedEdges) YIELD node, rank SET node.katz = rank;
```

Comment:
This test only appears to call `.update`; consider adding a different test where KC values aren't saved as node attributes, but retrieved by `.set` and `.get`.